app/views/welcome/index.html.erb

```diff
@@ -96,4 +96,11 @@
       <%end%>
     </div>
     <% end %>
-  </div>
\ No newline at end of file
+  </div>
+
+<script type="module">
+  // Make the AJAX call for version information explicitly on this page
+  // (and for now only on this page) so that we don't make extra calls
+  // to Mediaflux for every page loaded.
+  displayMediafluxVersion();
+</script>
```

Comment:
This is really neat...by using `type="module"` we can execute a function defined in `application.js`

(the function is in fact defined in `mediafluxVersion.js` but it is imported and made available via `application.js`)